sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -141,7 +141,7 @@ export class TestTracer extends NoOpTracer {
     }
 
 // @public
-export const enum TraceFlags {
```

Comment:
how did this get changed?

Reply:
I ran `rushx lint:fix` then `rushx build` 🤔

Reply:
This comes from the rule `@azure/azure-sdk/ts-no-const-enums`. I will ignore this rule here.